docker/vm_boot_images/Dockerfile

```diff
@@ -22,6 +22,9 @@ WORKDIR /app
 # Basic setup
 RUN ./ubuntu.sh
 
+# Point any MLflow tracking hooks at the main MLflow instance on Cloud Run
+ENV MLFLOW_TRACKING_URI='https://mlflow-783282864357.us-central1.run.app'
```

Comment:
if we are worried about this being visible publicly, we can take it out of the dockerfile and pull this as part of the bash script